### DISCUSSION TITLE
What is GAOKAO-2023?

### DISCUSSION THREAD

**Discussion by zhimin-z** [open-compass/opencompass#617] [Q&A] [answered]
https://opencompass.org.cn/dataset-detail/GAOKAO-2023

**Comment by yingfhu** [comment]
> GAOKAO-2023 is the collection of 2023 Chinese National College Entrance Examination (GaoKao) questions collected and created by Shanghai AI Laboratory. It currently consists of approximately 500 multiple-choice questions from various subjects.

more details can refers to config https://github.com/open-compass/opencompass/blob/main/configs/datasets/GaokaoBench/GaokaoBench_gen_5cfe9e.py and follow our tutorial to get the dataset

**Reply by zhimin-z** [reply]
Then what is the difference between GaokaoBench and Gaokao-2023?
![image](https://github.com/open-compass/opencompass/assets/8592144/ae29f07d-1fb2-452c-8ab4-54ea27fe6979)

**Reply by yingfhu** [reply, marked as answer]
Sorry for the confusion, the GaokaoBench2023 is not released yet, our config data is for GaokaoBench which includes questions in 2010-2022
